Glossary.adoc

```diff
@@ -1,5 +1,17 @@
 // Here goes all the term definitions the user might need to reference while reading the manual.
-// We need to figure out a way to sort the items here alphabetically
-// https://github.com/zynaddsubfx/user-manual/issues/6
 
-NOTE: Here by glossary!
\ No newline at end of file
+// Filter, sync, distortion, fade, pan
+// LFO, FM, AM, Bandwith, Additive/Subtractive synthesis, ADSR, Envelope and Harmonic
+
+// Ideally they're more naturally introduced in the text
+
+[glossary]
+Key:: A possible input to Zyn-Fusion, with a specific notation (eg. A4, G#2, etc). Practically, it's the same thing as a note.
+Patch:: A configuration of ZynAddSubFX the user makes or loads up. You can create patches for drums, strings, pads, or something entirely different! Other softsynths often call a patch a preset.
+Portamento:: A pitch slide from one note to another.
+Timbre:: The percieved quality of a sound. Different oscillators produce different timbres.
+VU meter:: A meter that displays the average level of an audio signal.
+Voice:: A single instance of sound a synth can produce. If you are limited to three voices, you can only play three notes at one time and expect to hear them in the final sound.
```

Comment:
That's the primary usage, but same as the last comment be aware that this gets overloaded to include a smaller contributing element within the add synth engine.

Reply:
Could you elaborate on this? What changes would you propose?

Reply:
For definitions that have multiple usages I'd mention the options in the definition. e.g. voice in most synths is typically an instance of something producing a sound, but in zyn it also refers to a piece of add synth where each voice in an add synth instance features one oscillator and one modulating oscillator.